engine/src/execution_graph/logic_controllers/LogicalProject.cpp

```diff
@@ -289,7 +290,12 @@ std::unique_ptr<cudf::column> evaluate_string_functions(const cudf::table_view &
                 computed_end_column = cudf::make_column_from_scalar(*end_scalar, table.num_rows());
                 end_column = computed_end_column->view();
             }
-
+            cudf::numeric_scalar<int32_t> start_zero(0);
```

Comment:
Before you do this operation, please check to see if there are any nulls you need to replace. Otherwise anytime there are no nulls, then this is an extra operation and allocation you dont need to do

Reply:
good catch ty!

Reply:
done